test/test_pipeline_baseline.py

```diff
@@ -106,24 +107,50 @@ def _test_python_baseline(task_name, test_name, tmpdir):
         os.makedirs(final_output_dir)
     with open(os.path.join(final_output_dir, 'fake_output_api.py'), 'w'):
         pass
+    extra_kwargs = {
+        'publish_server_url': 'https://example-site.exampledomain.com/',
+        'publish_uname': 'example-user'}
 
-    _test_baseline(task_name, test_name, 'python', output_dir)
+    _test_baseline(task_name, test_name, 'python', output_dir, extra_kwargs)
 
 
-def test_python_grpc_client_baseline(tmpdir):
-    _test_python_baseline('PythonGrpcClientPipeline',
-                          'python_grpc_client_pipeline',
-                          tmpdir)
+def _test_python_pub_baseline(task_name, test_name, tmpdir):
+    output_dir = str(tmpdir)
+
+    # Create an empty 'fake_output_api.py' in the output_dir. Do not invoke
+    #  'touch' command with subprocess.call() because it's mocked.
+    final_output_dir = os.path.join(output_dir, 'library-v1-gapic-gen-python')
+    if not os.path.exists(final_output_dir):
```

Comment:
the os code comes from the method _test_python_baseline in the current master branch (it was written by jun according to blame). I created _test_python_pub_baseline and _test_python_nopub_baseline based on _test_python_baseline, so I used this code. Should I figure out how to avoid manpiulating the filesystem?